pandas/tests/test_index.py

```diff
@@ -53,6 +53,14 @@ def test_pickle_compat_construction(self):
         # need an object to create with
         self.assertRaises(TypeError, self._holder)
 
+    def test_shift_index(self):
+        # err8083 test the base class for shift
+        idx = self.create_index()
+        self.assertRaises(NotImplementedError, idx.shift, 1)
+
+        result = idx.shift(periods=2, freq=2)
```

Comment:
take out this call right here as it will fail by definition (leave the ``assertRaises``)